### DISCUSSION TITLE
version `GLIBCXX_3.4.30' not found

### DISCUSSION THREAD

**Discussion by HubKing** [VOICEVOX/voicevox#1550] [Q&A] [answered]
UbuntuではなくArch系列でも使えますか? CPU, tar-gzバージョンをダウンロードしたんですが、`./voicevox`を実行すると

> voicevox-linux-cpu-0.14.7/VOICEVOX/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by /usr/lib/libicuuc.so.73)

のエラーが発生して起動できません。

**Comment by Hiroshiba** [comment, marked as answer]
@HubKing https://github.com/VOICEVOX/voicevox/issues/1237

**Reply by HubKing** [reply]
AURにある、誰かが作っておいた[AppImageバージョン](https://aur.archlinux.org/packages/voicevox-appimage)をインストールしたら使えるようになりました。
